test/JDBC/upgrade/13_5/schedule

```diff
@@ -242,3 +242,4 @@ BABEL-3655
 sys-table_types_internal
 BABEL-CHECK-CONSTRAINT-before-14_6
 sys_server_principals_dep_for_13_x
+TestVariableDataLength
```

Comment:
Let's not add this test here since the error messages are changing in 15 compared to 13.

Reply:
Alright